### PR TITLE
feat: seam governance

### PR DESCRIPTION
**Pull request by fredwes**

### CODE REVIEW

File: pages/governance/index.governance.tsx [hunk 1]
```diff
@@ -1,58 +1,57 @@
...
+// import StyledToggleButton from 'src/components/StyledToggleButton';
+// import StyledToggleButtonGroup from 'src/components/StyledToggleButtonGroup';
 import { GovDelegationModal } from 'src/components/transactions/GovDelegation/GovDelegationModal';
 import { MainLayout } from 'src/layouts/MainLayout';
 import { GovernanceTopPanel } from 'src/modules/governance/GovernanceTopPanel';
-import { ProposalsList } from 'src/modules/governance/ProposalsList';
 import { UserGovernanceInfo } from 'src/modules/governance/UserGovernanceInfo';
-import { Ipfs, IpfsType } from 'src/static-build/ipfs';
-import { CustomProposalType, Proposal } from 'src/static-build/proposal';
+//import { Ipfs, IpfsType } from 'src/static-build/ipfs';
```
Comment (kitanovicd) [inline pages/governance/index.governance.tsx:10]: Is there any specific reason we are commenting this lines and not deleting?

File: pages/governance/index.governance.tsx [hunk 2]
```diff
@@ -1,58 +1,57 @@
...
+//import { CustomProposalType, Proposal } from 'src/static-build/proposal';
 import { useRootStore } from 'src/store/root';
 
 import { ContentContainer } from '../../src/components/ContentContainer';
 
-export const getStaticProps = async () => {
-  const IpfsFetcher = new Ipfs();
-  const ProposalFetcher = new Proposal();
+// export const getStaticProps = async () => {
```
Comment (kitanovicd) [inline pages/governance/index.governance.tsx:16]: Same here

File: pages/governance/index.governance.tsx [hunk 3]
```diff
@@ -1,58 +1,57 @@
...
-        title: ipfs.title,
-        id: ipfs.id,
-        originalHash: ipfs.originalHash,
-        shortDescription: ipfs.shortDescription || '',
-      },
-      proposal,
-      prerendered: true,
-    };
-  });
+//   const proposals = [...Array(ProposalFetcher.count()).keys()].map((id) => {
```
Comment (kitanovicd) [inline pages/governance/index.governance.tsx:20]: Same

**Review by ry-animal — APPROVED** [review]
LGTM, nothing stands out - light scroll through lint/renamed files but the meat looks good.

**Review by kitanovicd — APPROVED** [review]

**Pull request merged**